src/parameterizations/lateral/MOM_internal_tides.F90

```diff
@@ -993,7 +1221,10 @@ subroutine itidal_lowmode_loss(G, GV, US, CS, Nb, Rho_bot, Ub, En, TKE_loss_fixe
         frac_per_sector = En(i,j,a,fr,m)/En_tot
         TKE_loss(i,j,a,fr,m) = frac_per_sector*TKE_loss_tot           ! [R Z3 T-3  ~> W m-2]
         loss_rate = TKE_loss(i,j,a,fr,m) / (En(i,j,a,fr,m) + En_negl) ! [T-1 ~> s-1]
-        En(i,j,a,fr,m) = En(i,j,a,fr,m) / (1.0 + dt*loss_rate)
+        En_b = En(i,j,a,fr,m)
+        En_a = En(i,j,a,fr,m) / (1.0 + dt*loss_rate)
+        TKE_loss(i,j,a,fr,m) = (En_b - En_a) / dt ! overwrite with exact value
```

Comment:
Instead of dividing by `dt` inside of loops, set a variable `I_dt = 1.0 / dt` outside of the loops and then multiply by that.   Most compilers will do this anyway at a certain level of optimization, but if we do it ourselves, the optimized and non-optimized (debugging) code are more likely to give the same answers.

Reply:
done